pygments/lexers/asc.py

```diff
@@ -0,0 +1,51 @@
+"""
+    pygments.lexers.asc
+    ~~~~~~~~~~~~~~~~~~~
+
+    Lexer for various ASCII armored files.
+
+    :copyright: Copyright 2021 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+import re
+
+from pygments.lexer import RegexLexer, bygroups
+from pygments.token import Comment, Generic, Name, Operator, Text, String
+
+__all__ = ['AscLexer']
+
+
+class AscLexer(RegexLexer):
+    """
+    Lexer for ASCII armored files, containing `-----BEGIN/END ...-----` wrapped base64 data.
+
+    .. versionadded:: 2.10
+    """
+    name = 'ASCII armored'
+    aliases = ['asc', 'pem']
+    filenames = [
+        '*.asc',  # PGP; *.gpg, *.pgp, and *.sig too, but those can be binary
+        '*.pem',  # X.509; *.cer, *.crt, *.csr, and key etc too, but those can be binary
+        'id_dsa', 'id_ecdsa', 'id_ecdsa_sk', 'id_ed25519', 'id_ed25519_sk', 'id_rsa',  # SSH private keys
+    ]
+    mimetypes = ['application/pgp-keys', 'application/pgp-encrypted', 'application/pgp-signature']
+
+    flags = re.MULTILINE
+
+    tokens = {
+        'root': [
+            (r'\s+', Text),
```

Comment:
That should be whitespace, right?

Reply:
I don't know, seems a lot of lexers mark whitespace as Text, and significantly fewer as Whitespace.

```shellsession
$ git grep -E "'\\\s[+*]', Text\)" | wc -l
272
$ git grep -E "'\\\s[+*]', .*Whitespace" | wc -l
33
```

But I'm fine either way, just let me know.

Reply:
Yeah, that's because we haven't paid enough attention to this previously :)

Reply:
Ok, changed to Whitespace.